unity-client/Assets/Scripts/MainScripts/DCL/Controllers/PlayerAvatarController/PlayerAvatarController.cs

```diff
@@ -1,9 +1,11 @@
-﻿using System.Runtime.CompilerServices;
 using DCL;
+using DCL.Interface;
 using UnityEngine;
 
 public class PlayerAvatarController : MonoBehaviour
 {
+    private const string LOADING_WEARABLES_ERROR_MESSAGE = "There was a problem loading some wearables";
```

Comment:
I think that this is not informative enough. Can we give more information in the message?

Something like 
"Oh, oh... Some wearables weren't able to load, you could see some people without the wearables, please {insert action to see wearables} to see them correctly

Reply:
Same as before... I'm currently confirming with @olavra the definitive messages for the user feedback.